### ISSUE TITLE
Dragging items in the NetworkList only works every other attempt in Safari

### ISSUE THREAD

**Issue by itsjohncs** [thelounge/thelounge#4327]
This is caused by the bug SortableJS/Sortable#1571. While Sortable.js has since fixed the problem, [Vue.Draggable has not upgraded its dependencies in awhile](https://github.com/SortableJS/Vue.Draggable/issues/1080) and it's unclear when they will. I see a few fine options:

1. Force Vue.Draggable to use a newer Sortable.js via [Yarn's Selective Dependency Resolution](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/) feature.
2. Maintain a fork of Vue.Draggable with upgraded Sortable.js and depend on it.
3. Send a PR to Vue.Draggable and hope for the best.

I think we should do 1. But there's an issue: yarn will emit a warning when installing.

```
$ yarn install
yarn install v1.22.15
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
warning Resolution field "sortablejs@1.14.0" is incompatible with requested version "sortablejs@1.10.2"
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
[5/5] 🔨  Building fresh packages...
success Saved lockfile.
✨  Done in 4.47s.
```

The only way to suppress the warning outside of always piping `yarn install` through something like `sed` is the [`logFilters` configuration available in Yarn 2](https://yarnpkg.com/configuration/yarnrc#logFilters) (we'd need to [upgrade](https://yarnpkg.com/getting-started/migration) which may not be trivial).

I think we ought to go ahead and upgrade to Yarn 2. Does anyone think it would be best to stay on Yarn 1?

**Comment by itsjohncs** [comment]
PR opened in Vue.Draggable to update SortableJS dependency (https://github.com/SortableJS/Vue.Draggable/pull/1085)

**Comment by itsjohncs** [comment]
#4330 fixes this.

**Comment by itsjohncs** [comment]
Fixed by #4330
